src/components/workers-page/workers-tab/enhanced-table-header-component.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import TableCell from "@material-ui/core/TableCell";
+import TableSortLabel from "@material-ui/core/TableSortLabel";
+import { Button } from "../../common-components";
+import { WorkerData } from "../../../common-models/worker-info.model";
+import { Order } from "../../../helpers/array.helper";
+import { createStyles, makeStyles } from "@material-ui/core/styles";
+import ScssVars from "../../../assets/styles/styles/custom/_variables.module.scss";
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    label: {
+      color: ScssVars.primary,
+      fontWeight: "normal",
+      fontSize: ScssVars.fontSizeBase,
+      fontFamily: ScssVars.fontFamilyPrimary,
+      letterSpacing: ScssVars.headingLetterSpacing,
+    },
+    activeLabel: {
+      color: ScssVars.primary,
+      fontWeight: "bold",
+      fontSize: ScssVars.fontSizeBase,
+      fontFamily: ScssVars.fontFamilyPrimary,
+      letterSpacing: ScssVars.headingLetterSpacing,
+    },
+    visuallyHidden: {
+      border: 0,
+      clip: "rect(0 0 0 0)",
+      height: 1,
+      margin: -1,
+      overflow: "hidden",
+      padding: 0,
+      position: "absolute",
+      top: 20,
+      width: 1,
+    },
```

Comment:
mógłbyś napisać o co tu chodziło, może da się jakoś to uprościć?

Reply:
A to jest właśnie gotowiec z materiala. Działa fajnie to się nie zagłębiałem, ale mogę zrobić research XD

Reply:
Ten kod nie był potrzebny, ukrywał on labelkę o kierunku sortowania, ale my jej w ogóle nie chcemy.